userspace/libscap/CMakeLists.txt

```diff
@@ -27,20 +27,35 @@ add_definitions(-DPLATFORM_NAME="${CMAKE_SYSTEM_NAME}")
 
 if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 	if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-		set(KBUILD_FLAGS "${SYSDIG_DEBUG_FLAGS}")
+		set(KBUILD_FLAGS "${LIBS_DEBUG_FLAGS}")
 	endif()
 
     if(NOT DEFINED PROBE_VERSION)
-        set(PROBE_VERSION "${SYSDIG_VERSION}")
+        set(PROBE_VERSION "${LIBS_VERSION}")
     endif()
     if(NOT DEFINED PROBE_NAME)
-        set(PROBE_NAME "sysdig-probe")
+        set(PROBE_NAME "falco")
     endif()
 
     if(NOT DEFINED PROBE_DEVICE_NAME)
-        set(PROBE_DEVICE_NAME "sysdig")
+        set(PROBE_DEVICE_NAME "falco")
     endif()
+
+	if(NOT DEFINED PROBE_ENV_VAR_NAME)
+		string(TOUPPER PROBE_DEVICE_NAME PROBE_ENV_VAR_NAME)
+	endif()
+endif()
+
+if(NOT DEFINED PROBE_ENV_VAR_NAME)
+	# fallback to default
+	set(PROBE_ENV_VAR_NAME "FALCO_BPF_PROBE") 
+endif()
+add_definitions(-DPROBE_ENV_VAR_NAME="${PROBE_ENV_VAR_NAME}")
+
+if(NOT DEFINED SCAP_HOST_ROOT_ENV_VAR_NAME)
+	set(PROBE_ENV_VAR_NAME "HOST_ROOT")
```

Comment:
Actually, `HOST_ROOT` (without prefix) is the default value we want (it's used by Falco) when `PROBE_ENV_VAR_NAME` is not defined.

In general, I wanted to let any libs' consumer free to set whatever they want since those values can be significantly different in other contexts.
For this reason, I avoided introducing a top-level PRODUCT_NAME  variable

Reply:
But you still want to set `SCAP_HOST_ROOT_ENV_VAR_NAME`, not `PROBE_ENV_VAR_NAME`, right?

Right now `PRODUCT_NAME` exists and is called `PROBE_NAME` (or maybe `PROBE_DEVICE_NAME`) whether you want it or not ;)

I'm not saying you should hardcode the policy to generate all these variables from PRODUCT_NAME (or rather, hardcode whatever you wish, but leave a way to supply your own)

Reply:
You're right! I didn't notice that. I will fix it. Thank you!